operator-common/src/main/java/io/strimzi/operator/common/operator/resource/DeploymentConfigOperator.java

```diff
@@ -72,9 +73,35 @@ protected Integer currentScale(String namespace, String name) {
     private boolean isObserved(String namespace, String name) {
         DeploymentConfig dep = get(namespace, name);
         if (dep != null)   {
-            return dep.getMetadata().getGeneration().equals(dep.getStatus().getObservedGeneration());
+            // Get the roll out status
+            //     => Sometimes it takes OCP some time before the generations are updated.
+            //        So we need to check the conditions in addition to detect such situation.
+            boolean rollOutNotStartedYet = false;
```

Comment:
Just to avoid confusion around double negatives, might it be worth changing this to 
```
rollOutStarted = true
```

As we assume it has started unless the condition states otherwise

Reply:
I changed it to not use the double negative.